components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Data Feeds"
+description = "Exporting Node Data from Chef Client Runs to External Data Collectors"
+date = 2020-05-05T13:19:02-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "settings"
+    weight = 10
++++
+
+## About Automate Data Feeds
+
+The Data Feed service is used to send node data to a 3rd party service. This can be useful when updating Configuration Management Databases, external Security Dashboards and ITSM platforms.
```

Comment:
@davymcaleer Is an endpoint the only way for Data Feed to send node data to a 3rd party service?

Reply:
See line 33 for clarification